apps/ledger-live-mobile/src/newArch/features/Web3Hub/components/ManifestsList/index.tsx

```diff
@@ -38,9 +39,8 @@ const renderItem = ({
   return <ManifestItem manifest={item} onPress={extraData} />;
 };
 
-export default function ManifestsList({ navigation, onScroll, pb = 0 }: Props) {
+export default function ManifestsList({ navigation, onScroll, title, pt = 0, pb = 0 }: Props) {
```

Comment:
Not a big fan of passing style this way, can we use spreading ?

Reply:
You loose performances by spreading and changing refs all the time
I also don't want people to pass more props/styles than what is needed, if I could do this cleaner no props for style would be needed at all but in this case you still need some control to share the logic between different screens
Spreading hides what you need and expect

Reply:
Alright